azurerm/internal/services/containers/tests/kubernetes_cluster_data_source_test.go

```diff
@@ -500,7 +499,11 @@ func testAccDataSourceAzureRMKubernetesCluster_addOnProfileAzurePolicy(t *testin
 					testCheckAzureRMKubernetesClusterExists(data.ResourceName),
 					resource.TestCheckResourceAttr(data.ResourceName, "addon_profile.0.azure_policy.#", "1"),
 					resource.TestCheckResourceAttr(data.ResourceName, "addon_profile.0.azure_policy.0.enabled", "true"),
-					resource.TestCheckResourceAttr(data.ResourceName, "addon_profile.0.azure_policy.0.version", "v2"),
+					// === RUN   TestAccAzureRMKubernetes_all/datasource/addOnProfileAzurePolicy
+					//    testing.go:684: Step 0 error: Check failed: Check 4/4 error: azurerm_kubernetes_cluster.test: Attribute 'addon_profile.0.azure_policy.0.version' not found
+					// It seems like it doesn't return version property anymore?
```

Comment:
entirely possibly, the transition period wasn't that long so maybe we need to default this in the response now?